experimental/PropertyDDS/packages/property-inspector-table/package.json

```diff
@@ -10,31 +10,34 @@
   },
   "license": "MIT",
   "author": "Microsoft and contributors",
-  "main": "index.js",
-  "types": "dist/types/index.d.ts",
+  "main": "dist/src/index.js",
+  "module": "lib/src/index.js",
+  "types": "dist/src/index.d.ts",
   "files": [
-    "dist/lib",
-    "dist/types",
-    "index.js"
+    "dist/src",
+    "dist/assets",
+    "lib/src",
+    "lib/assets"
   ],
   "scripts": {
     "build": "concurrently npm:build:compile npm:lint",
-    "build:compile": "concurrently npm:tsc npm:build:webpack",
-    "build:full": "npm run build",
-    "build:full:compile": "npm run build:compile",
-    "build:webpack": "npm run webpack",
+    "build:compile": "concurrently npm:tsc npm:build:esnext npm:webpack:svgs",
+    "build:esnext": "tsc --project ./tsconfig.esnext.json",
+    "build:full": "concurrently npm:build npm:webpack",
```

Comment:
since the typescript imports the webpacked assets, is this safe to run concurrently?

Reply:
I'm gonna remove the webpack build scripts, will have only the one for building svgs one.